packages/kit-bg/src/vaults/impls/stellar/Vault.ts

```
@@ -0,0 +1,1779 @@
+/* eslint-disable spellcheck/spell-checker */
+import BigNumber from 'bignumber.js';
+import { md5 } from 'js-md5';
+import { isEmpty, isNaN, orderBy } from 'lodash';
```

Biome (2.1.2)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)